scripts/visualize/visualize.hurricane_map.R

```diff
@@ -44,16 +44,26 @@ visualize_hurricane_map <- function(viz, height, width, mode, ...){
   g.rain <- xml_add_child(non.geo.top, 'g', id='legend', transform=sprintf("translate(10,%s)", as.numeric(vb[4])-50))
   
   # lower left legend:
-  xml_add_child(g.rain, 'text', "Total rainfall amount (inches)", class='svg-text legend-text', dy="-1em")
+  xml_add_child(g.rain, 'text', "Total rainfall amount (inches)", class='svg-text legend-text', dy="-1em",
+                transform="translate(0,35)")
+  g.gage_box <- xml_add_child(g.rain, 'rect', x = "0", y = "-52", height = "55", 
```

Comment:
instead of specifying `fill: #` here, it is usually better to give this group a class, so the style stuff can be changed in css (and the visualize step doesn't need to be run to experiment w/ styles)